tests/test_pytorch-notebook.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+import importlib
+import sys
+import os
+
+packages = [
+    # machine learning stuff
+    "pytorch_lightning",
+    "skimage",
+    "sklearn",
+    "torch",
+    "torchgeo",
+    # cupy import fails unless on GPU-enabled node:
+    #'cupy', #libcuda.so.1: cannot open shared object file: No such file or directory
```

Comment:
```suggestion
```

Let's remove this comment. It's just residue from old stuff in ml-image.

Reply:
Could add a suggestion as well? Because of the mkl and nomkl conflicts, this is sort of worrying... 

Try a test for the underlying blas to make sure things are set up correctly:

Usually it should be MKL (it's pulling MKL from conda-forge):

```python
import torch
assert torch.__config__.show()[torch.__config__.show().find('BLAS_INFO')+10:torch.__config__.show().find('BLAS_INFO')+13] == "mkl"
```

Reply:
Ok, done both in 3acef32a4f614726a502068446bd6c93e891dfb9. Let's see if the CI passes.

Reply:
Yep, works like a charm: `tests/test_pytorch-notebook.py::test_torch_uses_mkl PASSED               [100%]` https://github.com/pangeo-data/pangeo-docker-images/runs/6202609085?check_suite_focus=true#step:7:41